### ISSUE TITLE
RIMES dataset link might be not available now?

### ISSUE THREAD

**Issue by ThunderVVV** [FactoDeepLearning/VerticalAttentionOCR#14]
I can't open the link of http://www.a2ialab.com/doku.php?id=rimes_database:start
Is there something wrong with the website?

**Comment by FactoDeepLearning** [comment]
Indeed, I assume this dataset is no longer provided by A2ia. And unfortunately, I don't have the rights to share it.
